config/locales/client.en.yml

```diff
@@ -1838,6 +1838,26 @@ en:
         title: "Topic timeline"
         content: "Scroll quickly through a post using the topic timeline."
 
+      post_menu:
+        title: "Post menu"
+        content: "See how else you can interact with the post by clicking the three dots!"
```

Comment:
Can we check (somehow) whether the three dots menu is present? Because if a site doesn't have that menu, this copy won't be accurate.

Reply:
The tip is tied to the three dots button. If it does not exist then the tip does not show at all.